pkg/agent/templates/templates_windows.go

```diff
@@ -0,0 +1,174 @@
+// +build windows
+
+package templates
+
+import (
+	"bytes"
+	"strings"
+	"text/template"
+)
+
+const ContainerdConfigTemplate = `
+version = 2
+root = "{{ replace .NodeConfig.Containerd.Root }}"
+state = "{{ replace .NodeConfig.Containerd.State }}"
+plugin_dir = ""
+disabled_plugins = []
+required_plugins = []
+oom_score = 0
+
+[grpc]
+  address = "{{ replace .NodeConfig.Containerd.Address }}"
+  tcp_address = ""
+  tcp_tls_cert = ""
+  tcp_tls_key = ""
+  uid = 0
+  gid = 0
+  max_recv_message_size = 16777216
+  max_send_message_size = 16777216
+
+[ttrpc]
+  address = ""
+  uid = 0
+  gid = 0
+
+[debug]
+  address = ""
+  uid = 0
+  gid = 0
+  level = ""
+
+[metrics]
+  address = ""
+  grpc_histogram = false
+
+[cgroup]
+  path = ""
+
+[timeouts]
+  "io.containerd.timeout.shim.cleanup" = "5s"
+  "io.containerd.timeout.shim.load" = "5s"
+  "io.containerd.timeout.shim.shutdown" = "3s"
+  "io.containerd.timeout.task.state" = "2s"
+
+[plugins]
+  [plugins."io.containerd.gc.v1.scheduler"]
+    pause_threshold = 0.02
+    deletion_threshold = 0
+    mutation_threshold = 100
+    schedule_delay = "0s"
+    startup_delay = "100ms"
+  [plugins."io.containerd.grpc.v1.cri"]
+    disable_tcp_service = true
+    stream_server_address = "127.0.0.1"
+    stream_server_port = "0"
+    stream_idle_timeout = "4h0m0s"
+    enable_selinux = false
+    selinux_category_range = 0
+    sandbox_image = "mcr.microsoft.com/oss/kubernetes/pause:1.4.0"
```

Comment:
Should be same as on linux:
```
sandbox_image = "{{ .NodeConfig.AgentConfig.PauseImage }}"
```

Which implies that the default for that should be figured out for the `NodeConfig.AgentConfig`